app/controllers/services/govbox_controller.rb

```diff
@@ -28,6 +28,31 @@ def register_step1
   end
 
   def register_step2
+    @statutory =
+      begin
+        response = RestClient::Resource.new("#{ENV.fetch('AUTOFORM_URL')}/api/corporate_bodies/search", timeout: 10)
+          .get(params: { q: "cin:#{params[:cin]}", private_access_token: ENV.fetch('AUTOFORM_PRIVATE_ACCESS_TOKEN') } )
+
+        json = JSON.parse(response.body).first
+        return nil unless json
+
+        json.deep_symbolize_keys.fetch(:statutory).each_with_index.map do |person, index|
```

Comment:
urcite potrebujeme index? (aj tu aj neskor)

Reply:
no chcel som to kvoli html select options a tie potom potrebujeme referencovat, ze ktory je selected, pozri tu  https://github.com/slovensko-digital/www.ecosystem/pull/76/files#diff-f30b29d67138964731ffdbca4d56dad7R20

Reply:
rozmyslam, ze by som tu drzal data tak ako pridu a vybral si z toho co treba pripadne to naformatoval az vo views